.packit.yaml

```diff
@@ -18,10 +19,10 @@ downstream_package_name: foreman
 
 actions:
   post-upstream-clone:
-    - "wget https://raw.githubusercontent.com/theforeman/foreman-packaging/rpm/develop/packages/foreman/foreman/foreman.spec -O foreman.spec"
-    - "wget https://raw.githubusercontent.com/theforeman/foreman-packaging/rpm/develop/packages/foreman/foreman/foreman.logrotate -O foreman.logrotate"
-    - "wget https://raw.githubusercontent.com/theforeman/foreman-packaging/rpm/develop/packages/foreman/foreman/foreman.cron.d -O foreman.cron.d"
-    - "wget https://raw.githubusercontent.com/theforeman/foreman-packaging/rpm/develop/packages/foreman/foreman/foreman.tmpfiles -O foreman.tmpfiles"
+    - "wget https://raw.githubusercontent.com/theforeman/foreman-packaging/rpm/3.7/packages/foreman/foreman/foreman.spec -O foreman.spec"
+    - "wget https://raw.githubusercontent.com/theforeman/foreman-packaging/rpm/3.7/packages/foreman/foreman/foreman.logrotate -O foreman.logrotate"
+    - "wget https://raw.githubusercontent.com/theforeman/foreman-packaging/rpm/3.7/packages/foreman/foreman/foreman.cron.d -O foreman.cron.d"
+    - "wget https://raw.githubusercontent.com/theforeman/foreman-packaging/rpm/3.7/packages/foreman/foreman/foreman.tmpfiles -O foreman.tmpfiles"
   get-current-version: "sed 's/-develop//' VERSION"
```

Comment:
I debated if we need to suffix anything to indicate it's going to be VERSION + 1, but that it's not the final one of that. I want to check what ends up being built. If it uses the prerelease macros to set the git version that it's going to be correct.

Reply:
nah, packit adds some suffix, which makes you end up with `3.7.0-1.20230717143634628839.pr9775.1.g96e51e9fc.el8` right now, and that sounds sane to me